### PR TITLE
[service.subtitles.pipocas] 1.0.0

### PR DESCRIPTION
**Pull request by highlandr**
Compatibility from Gotham to Matrix thanks to Leinad4Mind and turtuga.

### CODE REVIEW

**Comment by highlandr** [comment]
@enen92 can you take a look ?

**Comment by basrieter** [comment]
You addon.xml is not correct. Please fix it.

**Comment by highlandr** [comment]
@enen92 @basrieter ?:)
Thanks.

File: service.subtitles.pipocas/addon.xml [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
-<addon id="service.subtitles.pipocas" name="Pipocas.tv" version="0.1.5" provider-name="HiGhLaNdeR">
+<addon id="service.subtitles.pipocas" name="Pipocas.tv" version="1.0.0" provider-name="HiGhLaNdeR, Leinad4Mind">
   <requires>
     <import addon="xbmc.python" version="2.14.0"/>
+    <import addon="script.module.requests" version="2.12.4"/>
   </requires>
-  <extension point="xbmc.subtitle.module" library="service.py" />
+  <extension point="xbmc.subtitle.module" library="service.py"/>
   <extension point="xbmc.addon.metadata">
     <summary lang="en">Pipocas.tv</summary>
```
Comment (enen92) [inline service.subtitles.pipocas/addon.xml]: en_GB

File: service.subtitles.pipocas/addon.xml [hunk 2]
```diff
@@ -1,21 +1,26 @@
...
+    <summary lang="pt">Pipocas.tv</summary>
```
Comment (enen92) [inline service.subtitles.pipocas/addon.xml]: pt_PT

File: service.subtitles.pipocas/addon.xml [hunk 3]
```diff
@@ -1,21 +1,26 @@
...
+    <summary lang="ptbr">Pipocas.tv</summary>
```
Comment (enen92) [inline service.subtitles.pipocas/addon.xml]: pt_BR

File: service.subtitles.pipocas/addon.xml [hunk 4]
```diff
@@ -1,21 +1,26 @@
...
-    <description lang="pt">Procura e download de legendas do site Pipocas.tv. Por favor verifica as configurações do add-on antes de usar! Nota: O campo "Procura e sincroniza pela pasta do filme" em modo "AUTO" vai primeiro tentar procurar pelo nome da directoria "PAI" mas se a mesma não for uma release irá procurar pelo nome do ficheiro, recomenda-se que esteja em "AUTO". Se o filme ou série estiver na Biblioteca, o nome do filme\serie é da Biblioteca.</description>
-    <disclaimer lang="en">highlander@teknorage.com</disclaimer>
-    <disclaimer lang="pt">highlander@teknorage.com</disclaimer>
-    <language>pt en</language>
+    <description lang="pt">Procura e descarrega legendas do site Pipocas.tv. Por favor, verifica as definições do add-on antes de usar! Nota: O campo "Procura e sincroniza pela pasta do filme" em modo "AUTO" vai primeiro tentar procurar pelo nome da pasta "PAI" mas se a mesma não for uma release irá procurar pelo nome do ficheiro, recomenda-se que esteja em "AUTO". Se o filme ou série estiver na Biblioteca, o nome do filme/série é da Biblioteca.</description>
```
Comment (enen92) [inline service.subtitles.pipocas/addon.xml]: same as above

File: service.subtitles.pipocas/addon.xml [hunk 5]
```diff
@@ -1,21 +1,26 @@
...
+    <description lang="ptbr">Pesquisa e baixa legendas do site Pipocas.tv. Por favor, verifica as configurações do add-on antes de usar! Nota: O campo "Procura e sincroniza pela pasta do filme" em modo "AUTO" vai primeiro tentar procurar pelo nome do diretório "PAI" mas se a mesma não for uma release irá procurar pelo nome do arquivo, recomenda-se que esteja em "AUTO". Se o filme ou série estiver na Biblioteca, o nome do filme/série é da Biblioteca.</description>
+    <disclaimer lang="en">For bugs, requests or general questions please contact: highlander@teknorage.com</disclaimer>
+    <disclaimer lang="pt">Para bugs, pedidos ou perguntas, contacte: highlander@teknorage.com</disclaimer>
+    <disclaimer lang="ptbr">Para bugs, pedidos ou perguntas, contate: highlander@teknorage.com</disclaimer>
+    <language>pt en ptbr</language>
```
Comment (enen92) [inline service.subtitles.pipocas/addon.xml]: pt en

File: service.subtitles.pipocas/resources/language/English/strings.po [hunk 6]
```diff
@@ -0,0 +1,121 @@
+# Addon Name: Pipocas.tv
```
Comment (enen92) [inline service.subtitles.pipocas/resources/language/English/strings.po]: please delete the strings.mo file, it's not valid. Its strings.po

File: service.subtitles.pipocas/resources/language/Portuguese (Brazil)/strings.po [hunk 7]
```diff
@@ -0,0 +1,121 @@
+# Addon Name: Pipocas.tv
```
Comment (enen92) [inline service.subtitles.pipocas/resources/language/Portuguese (Brazil)/strings.po]: same for Portuguese Brazil, remove the .mo file

File: service.subtitles.pipocas/resources/language/Portuguese/strings.po [hunk 8]
```diff
@@ -0,0 +1,121 @@
+# Addon Name: Pipocas.tv
```
Comment (enen92) [inline service.subtitles.pipocas/resources/language/Portuguese/strings.po]: same for Portuguese, remove strings.mo

**Comment by highlandr** [comment]
@enen92 should be all fixed, hopefully.
Thanks.
Regards.

**Pull request merged**